frontend/src/components/InputText.js

```diff
@@ -1,8 +1,64 @@
 import React, {Component} from 'react';
+import InputTextItems from './InputTextItems'
+import './InputText.css'
 
 class InputText extends Component {
+    constructor(props) {
+        super(props);
+       
+        this.state = {
+          items: []
+        };
+       
+        this.addItem = this.addItem.bind(this);
+        this.deleteItem = this.deleteItem.bind(this);
+
+      }
+
+    addItem(e) {
+        if (this._inputElement.value !== "") {
+          var newItem = {
+            text: this._inputElement.value,
+            key: Date.now()
+          };
+       
+          this.setState((prevState) => {
+            return { 
+              items: prevState.items.concat(newItem) 
+            };
+          });
+         
+          this._inputElement.value = "";
+        }
+         
+        console.log(this.state.items);
+           
+        e.preventDefault();
+    }
+    deleteItem(key) {
+        var filteredItems = this.state.items.filter(function (item) {
+          return (item.key !== key);
+        });
+       
+        this.setState({
+          items: filteredItems
+        });
+    }
+    
     render() {
-        return (<h1>Text</h1>)
+        return (
+          <div className="todoListMain">
```

Comment:
1. Change the className to a name that describes this list, i.e ingredient
2. `Do not concatenate words and abbreviations in selectors by any characters (including none at all) other than hyphens, in order to improve understanding and scannability.`
https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters